### DISCUSSION TITLE
fedora swayfx  blur not working

### DISCUSSION THREAD

**Discussion by spambroo** [WillPower3309/swayfx#342] [Q&A]
i am currently using sway on my fedora 40 workstations... In sway session, i have installed swayfx from fedora copr but i cant able to do  blur stuffs.....but corner radius is working ....

 ~ sway -v
-   swayfx version 0.4 (based on sway 1.9.0)

**Comment by WillPower3309** [comment]
What's wrong with blur? Do you have transparent windows?

**Reply by ErikReider** [reply]
Closed as resolved so user error I guess? ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯
